kolibri/core/urls.py

```diff
@@ -4,12 +4,22 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django.conf.urls import url
+from kolibri.plugins import hooks
 
 from . import views
 
 app_name = 'kolibri'
 
 urlpatterns = [
-    url('^', views.IndexView.as_view(), name='index'),
+    url('^/$', views.IndexView.as_view(), name='index'),
```

Comment:
Probably not what you intended -- e.g. 404s on `http://127.0.0.1:8000/` but catches `http://127.0.0.1:8000//`. See the [APPEND_SLASH setting](https://docs.djangoproject.com/en/1.9/ref/settings/#append-slash).